pyvista/core/filters/data_set.py

```diff
@@ -300,6 +303,12 @@ def clip_scalar(dataset, scalars=None, invert=True, value=0.0, inplace=False, pr
         >>> clipped = dataset.clip_scalar(scalars="sample_point_scalars", value=100)
         >>> clipped.plot()
 
+        Get clipped meshes corresponding to the portions of the mesh above and below 100
```

Comment:
```suggestion
        Get clipped meshes corresponding to the portions of the mesh above and below 100.

```

We have to have a space here to get sphinx to render it correctly.